### PR TITLE
updates ref version 2

### PR DESCRIPTION
**Pull request by mikkelcodes**

### CODE REVIEW

File: packages/website/src/components/featuredTeam/index.js [hunk 1]
```diff
@@ -5,6 +5,7 @@ import { Title } from '../title';
 import * as Button from '../button';
 
 export const FeaturedTeam = ({ team, notitle, fallbackImg }) => {
+  console.log(team);
```
Comment (TheKnarf) [inline packages/website/src/components/featuredTeam/index.js:8]: Remove the console.log

File: packages/website/src/components/navigation/index.js [hunk 2]
```diff
@@ -2,29 +2,30 @@ import React from 'react';
...
+import { LargeLink, Subtitle, ListLink } from '../navItems';
 import useWindowDimensions from '../../hooks/useWindowDimensions';
 
 const ConditionalWrapper = ({ condition, wrapper, children }) =>
   condition ? wrapper(children) : children;
 
-export const Navigation = ({ open, toggleClose }) => {
+export const Navigation = ({ open, toggleClose, pages }) => {
   const { width } = useWindowDimensions();
+  console.log(pages);
```
Comment (TheKnarf) [inline packages/website/src/components/navigation/index.js:13]: Remove console.log

File: packages/website/src/components/navigation/index.js [hunk 3]
```diff
@@ -48,23 +49,38 @@ export const Navigation = ({ open, toggleClose }) => {
...
+            {pages &&
+              pages.map((page) => {
+                console.log(page);
+                return (
+                  <ListLink link={`/vi-tilbyr/${page.slug.current}`}>
+                    {page.heroHeading}
+                  </ListLink>
+                );
+              })}
+            {/* <ListLink link="/vi-tilbyr/c-utvikler">Java Utvikler</ListLink>
```
Comment (TheKnarf) [inline packages/website/src/components/navigation/index.js:83]: Can we remove this out commented code?

Comment (mikkelcodes) [reply packages/website/src/components/navigation/index.js:83]: I will save a copy of the file locally and remove all uncommented code

**Review by TheKnarf — APPROVED** [review]

**Pull request merged**